### PR TITLE
added Subscribing functionality to courses

### PR DESCRIPTION
**Pull request by Vitya11**

### CODE REVIEW

File: app/modules/core/views/modalView.js [hunk 1]
```diff
@@ -38,7 +39,14 @@ define(function(require) {
         },
         declinePopup: function(){
             this.$el.modal("hide");
+        },
+        showSuccessMesasage: function(mess){
+
+            $(".modal-dialog").html(this.successTemplate({successMessage: mess }));
+            //this.render();
```
Comment (okrasn) [inline app/modules/core/views/modalView.js]: Why commented line present?

File: app/modules/course/views/CourseView.js [hunk 2]
```diff
@@ -7,20 +7,63 @@ define(function(require) {
...
         },
-        subscribeDialog: function(ev){
+        subscribeDialog: function(e){
+            var _this = this;
+            CMS.ModalView.prototype.submitHandlerClick = function() {
+                $.ajax({
+                    url: _this.subscribeUrl,
+                    method: "POST",
+                    data: {id: this.model.id, name: this.model.get("name")},
+                    beforeSend: function(jqXHR){
```
Comment (okrasn) [inline app/modules/course/views/CourseView.js]: 1. Put ajax method to model. And use model method to send ajax.
2. `beforeSend` property put to `ajaxSetup` functionality

File: app/modules/course/views/CourseView.js [hunk 3]
```diff
@@ -7,20 +7,63 @@ define(function(require) {
...
+            };
+            this.subscribeModal.modalHeader = "Я підтверджую подачу заявки на курс:";
+            this.subscribeModal.submitButton = "Подати заявку";
+            this.subscribeModal.render();
+            this.subscribeModal.show();
+        },
+        unSubscribeDialog: function(e){
+            var _this = this;
+            CMS.ModalView.prototype.submitHandlerClick = function() {
+                $.ajax({
```
Comment (okrasn) [inline app/modules/course/views/CourseView.js]: Here is the same comments as to ajax method before

**Comment by Vitya11** [comment]
Done. I think so...

**Pull request merged**